sagar/data/Lab_1sagar.ipynb

```diff
@@ -0,0 +1,1314 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install mplfinance
</pre><p>Dont add this here. Add mplfinance in requirements.txt file (if it is not present already)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>